src/core/Snippets.ts

```diff
@@ -97,22 +97,9 @@ export class Snippets {
    * @returns Boolean indicating whether path has changed
    */
   private updateUserSnippetsPath(): boolean {
-    let newUserSnippetsDir: string | undefined;
-
-    if (isTesting()) {
-      newUserSnippetsDir = join(
-        this.graph.extensionContext.extensionPath,
-        "src",
-        "test",
-        "suite",
-        "fixtures",
-        "cursorless-snippets",
-      );
-    } else {
-      newUserSnippetsDir = workspace
-        .getConfiguration("cursorless.experimental")
-        .get<string>("snippetsDir");
-    }
+    const newUserSnippetsDir = ide().configuration.getOwnConfiguration(
+      "experimental.snippetsDir",
+    );
```

Comment:
Note that we don't need the check for `isTesting` along with hacked path.  We have proper mocking now